### PR TITLE
fixing upvote/downvote issues

### PR DESCRIPTION
**Pull request by sourya1995**
- added simple toggle for ensuring upvote/downvote is removed on subsequent click.
- please be aware that this is for the carousel ONLY.

### CODE REVIEW

File: ios/LinksByNK.xcodeproj/project.pbxproj
Comment (saransh12) [inline ios/LinksByNK.xcodeproj/project.pbxproj:1]: What are these changes? Why are they required?

File: ios/LinksByNK.xcodeproj/xcshareddata/xcschemes/LinksByNK.xcscheme
Comment (saransh12) [inline ios/LinksByNK.xcodeproj/xcshareddata/xcschemes/LinksByNK.xcscheme:1]: What are these changes? Why are they required?

File: ios/LinksByNK.xcodeproj/project.pbxproj
Comment (sourya1995) [reply ios/LinksByNK.xcodeproj/project.pbxproj:1]: I'm not sure but they seem to be iOS related files that may have been integrated when I pulled from main.

File: ios/LinksByNK.xcodeproj/xcshareddata/xcschemes/LinksByNK.xcscheme
Comment (sourya1995) [reply ios/LinksByNK.xcodeproj/xcshareddata/xcschemes/LinksByNK.xcscheme:1]: I'm not sure but they seem to be iOS related files that may have been integrated when I pulled from main.

**Review by Shahid-Tumbi — APPROVED** [review]

**Pull request merged**